src/components/summary/summarySection.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import healthData from '../../app/cohort_template';
+
+type Props = {
+    dataKey: string;
+}
+const MainSummarySection = ({ dataKey } : Props) => {
+    const summaryData = healthData[dataKey]?.summary;
```

Comment:
Nice job going ahead and making this responsive to the user's input 😎

Reply:
I made some quick edits with Tailwind for some styling pump. You don't have to match this, but it can be a good starting point: 
<img width="924" alt="image" src="https://github.com/dpericich/Health-Checklist/assets/53955965/08d19bcd-c9f0-465f-aac1-41401b7b8af5">